### PR TITLE
Implement SetRPM

### PR DESCRIPTION
**Pull request by randhid**

### CODE REVIEW

**Comment by randhid** [comment]
FYI @zaporter-work, when you start coding the motor RC card with the SetRPM endpoint.

File: fakemotor/motor.go [hunk 1]
```diff
@@ -89,21 +89,23 @@ func (f *fake) SetPower(ctx context.Context, power float64, extra map[string]int
 	return nil
 }
 
+func (f *fake) SetRPM(ctx context.Context, rpm float64, extra map[string]interface{}) error {
+	f.mu.Lock()
+	defer f.mu.Unlock()
```
Comment (randhid) [inline fakemotor/motor.go:94]: ah no - should remove.

Comment (zaporter-work) [reply fakemotor/motor.go:94]: probably

**Review by zaporter-work — COMMENTED** [review]
Nice!

**Pull request merged**